setup.py

```diff
@@ -216,8 +216,8 @@ def get_extensions():
 Numpy is a dependency for both.
 
 The documentation can be found
-`here <http://hgomersall.github.com/pyFFTW/>`_, and the source
-is on `github <https://github.com/hgomersall/pyFFTW>`_.
+`here <http://pyFFTW.github.io/pyFFTW/>`_, and the source
+is on `github <https://github.com/pyFFTW/pyFFTW>`_.
 '''
```

Comment:
Wasn't able to do that exactly as it didn't work with `conda` (needed to install `fftw`), but did something similar. Now it seems to be unable to find `gcc`? Kind of strange. 😕 Log below if you are interested.

ref: https://readthedocs.org/projects/jakirkhampyfftw/builds/5243045/